osmosis-1/osmosis-1.assetlist.json

```diff
@@ -2978,9 +2978,9 @@
       "type_asset": "ics20",
       "address": "juno1r4pzw8f9z0sypct5l9j906d47z998ulwvhvqe5xdwgy8wf84583sxwh0pa",
       "base": "ibc/6BDB4C8CCD45033F9604E4B93ED395008A753E01EECD6992E7D1EA23D9D3B788",
-      "name": "Racoon",
-      "display": "rac",
-      "symbol": "RAC",
+      "name": "Racoon.legacy",
+      "display": "rac.legacy",
+      "symbol": "RACx",
```

Comment:
Updates to the legacy RAC token should take place at the Chain Registry, and then those changes will be brought over automatically. Please make these changes to the Chain Registry